source/AI.cpp

```diff
@@ -2720,7 +2720,8 @@ void AI::AimTurrets(const Ship &ship, Command &command, bool opportunistic) cons
 			{
 				// Get the index of this weapon.
 				int index = &hardpoint - &ship.Weapons().front();
-				double offset = (hardpoint.HarmonizedAngle() - hardpoint.GetAngle()).Degrees();
+				double offset = (hardpoint.GetBaseAngle() + hardpoint.HarmonizedAngle()
```

Comment:
Do we really need to give a default where the turret aims at (and provide that as keyword)? (I would expect harmonized-forward if harmonized-forward is in range, otherwise normal forward if it is in range, and otherwise the center of the range).

Reply:
An appearance of a ship depends on the idle position of turrets.
It can create a turret like the second and the third turret of [HMS Dreadnought](https://en.wikipedia.org/wiki/File:HMS_Dreadnought_(1911)_profile_drawing.png) and to shoot an asteroid when mining as same as a current behavior. It points to the front, but possibly someone wants it points the back.

Reply:
This can be a nice feature to have, but I'm not sure this is required now. We can also limit the rotation of a turret without providing a default holding/idle position.